lib/reek/smells/utility_function.rb

```diff
@@ -53,11 +56,13 @@ def contexts # :nodoc:
       # @return [Array<SmellWarning>]
       #
       # :reek:FeatureEnvy
+      # :reek:TooManyStatements: { max_statements: 6 }
```

Comment:
Kind of sucks, but refactoring UtilityFunction is out of the scope for this PR and I'd rather address this separately.